src/MLJGLMInterface.jl

```diff
@@ -233,43 +272,45 @@ glm_fitresult(::LinearBinaryClassifier, fitresult) = fitresult[1]
 
 function MMI.fitted_params(model::GLM_MODELS, fitresult)
     result = glm_fitresult(model, fitresult)
-    coef = GLM.coef(result)
-    features = filter(name -> name != "(Intercept)", GLM.coefnames(result))
-    intercept = model.fit_intercept ? coef[end] : nothing
-    return (; features=features, coef=coef, intercept=intercept)
+    coef = coefs(result)
+    feature_names = copy(params(result).feature_names)
+    if model.fit_intercept
```

Comment:
Is there a reason for the key change `features` -> `feature_names`? I think in MLJ more generally `features` is standard (eg, transforms in MLJModels).

Reply:
Yeah, I'll fix that